sdcm/cluster.py

```diff
@@ -3678,26 +3708,27 @@ def verify_node_setup(start_time):
             cl_inst.update_db_binary(node_list, start_service=False)
             cl_inst.update_db_packages(node_list, start_service=False)
 
-        for node in node_list:
-            if isinstance(cl_inst, BaseScyllaCluster) \
-                    and not getattr(cl_inst, 'params', {}).get('use_legacy_cluster_init'):
-                init_nodes.append(node)
-                start_time = time.perf_counter()
-                node_setup(node)
+        with critical_node_setup_events():
```

Comment:
In general I'm also missing here an option to disable this, and leaving room for negative testing

example, if I want to test a flow that would end up with node failing boot, how would I call `cluster.add_nodes()` without enabling this feature

Reply:
o.k. just check what you are suggest here is working, and the parameter is pass down as expected

Reply:
o.k.